src/Billing/Billing/ChangeLog.md

```diff
@@ -20,7 +20,10 @@
 ## Upcoming Release
 
 ## Version 1.0.3
-* Updated assembly version of consumption cmdlets
+* Added `Get-AzBillingAccount` cmdlet
```

Comment:
please put release notes under
## Upcoming Release

and add "* Updated assembly version of consumption cmdlets" back under ## Version 1.0.3

Reply:
Sure! I have updated the ChangeLog and sent a new iteration.